deploy/infra/groups/data.bicep

```diff
@@ -123,7 +151,90 @@ module synapseIdentityForStorageAccess '../modules/storage-role-assignment.bicep
   ]
 }]
 
+module postgresqlServer '../modules/postgres.bicep' = {
+  name: '${namingPrefix}-postgres'
+  params: {
+    location: location
+    serverName: serverNameVar
+    administratorLogin: administratorLoginVar
+    administratorLoginPassword: postgresAdminLoginPassVar
+    skuCapacity: skuCapacity
+    skuName: skuName
+    skuSizeMB: skuSizeMB
+    skuTier: skuTier
+    skuFamily: skuFamily
+    postgresqlVersion: postgresqlVersion
+    backupRetentionDays: backupRetentionDays
+    geoRedundantBackup: geoRedundantBackup
+  }
+}
+
+resource postgresql_server_resource 'Microsoft.DBforPostgreSQL/servers@2017-12-01' existing = {
+  name: serverNameVar
+}
+
+resource open_postgres_firewall 'Microsoft.DBforPostgreSQL/servers/firewallRules@2017-12-01' = {
+  name: 'allowAll'
```

Comment:
we should revisit allowAll as default against the comment left above for End and Start IP addresses